samples/nrf9160/secure_boot/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Secure Boot"
+
+	config SB_BOOT_SILENTLY
```

Comment:
Will setting this to y, make sure the UART is disabled and not drawing any current? CC @Rallare

Reply:
I don't know, but probably that should be set using other Kconfig options.

Reply:
I agree that should not be the responsibility of how you setup the SPU however the `SB_BOOT_SILENTLY` should probably have a `depends on <Logging enabled>`. Mostly for one reason and that is logging could be disabled and it would boot silently anyways(hence UART is disabled etc). 

Another reason could be to make sure it doesn't inadvertently enables the UART(something it currently won't do since it's not doing any select).

Reply:
I kept this separate from logging on purpose, see: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/433#discussion_r265058250

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/433#discussion_r265058250 's reasoning is fine by me. It is an interesting observation nonetheless with regards to ROM footprint.